tests/testOpH5N5WriterBigDataset.py

```diff
@@ -0,0 +1,251 @@
+        assert numpy.all(n5_dataset[...] == self.testData.view(numpy.ndarray)[...])
+        hdf5File.close()
+        n5File.close()
+
+class TestOpH5N5WriterBigDataset_3(object):
+
+    def setUp(self):
+        self.graph = lazyflow.graph.Graph()
+        self.testDataH5FileName = 'bigH5TestData.h5'
+        self.testDataN5FileName = 'bigH5TestData.n5'
+
+        self.datasetInternalPath = 'volume/data'
+
+        # Generate some test data
+        self.dataShape = (1, 10, 128, 128, 1)
+        self.testData = vigra.VigraArray( self.dataShape, axistags=vigra.defaultAxistags('txyzc') ) # default vigra order this time...
+        self.testData[...] = numpy.indices(self.dataShape).sum(0)
+
+    def tearDown(self):
+        # Clean up: Delete the test file.
+        try:
+            os.remove(self.testDataH5FileName)
+            rmtree(self.testDataN5FileName)
+        except:
+            pass
+
+    def test_Writer(self):
+        # Create the h5 file
+        hdf5File = h5py.File(self.testDataH5FileName)
+        n5File = z5py.N5File(self.testDataN5FileName)
+
+        opPiper = OpArrayPiper(graph=self.graph)
+        opPiper.Input.setValue(self.testData)
+
+        # Force extra metadata onto the output
+        opPiper.Output.meta.ideal_blockshape = (1, 1, 0, 0, 1)
+        # Pretend the RAM usage will be really high to force lots of tiny blocks
+        opPiper.Output.meta.ram_usage_per_requested_pixel = 1000000.0
+        
+        h5_opWriter = OpH5N5WriterBigDataset(graph=self.graph)
+        n5_opWriter = OpH5N5WriterBigDataset(graph=self.graph)
+
+        # This checks that you can give a preexisting group as the file
+        h5_g = hdf5File.create_group('volume')
+        n5_g = n5File.create_group('volume')
+        h5_opWriter.h5N5File.setValue(h5_g)
+        n5_opWriter.h5N5File.setValue(n5_g)
+        h5_opWriter.h5N5Path.setValue("data")
+        n5_opWriter.h5N5Path.setValue("data")
+        h5_opWriter.Image.connect(opPiper.Output)
+        n5_opWriter.Image.connect(opPiper.Output)
+
+        # Force the operator to execute by asking for the output (a bool)
+        h5_success = h5_opWriter.WriteImage.value
+        n5_success = n5_opWriter.WriteImage.value
+        assert h5_success
+        assert n5_success
+
+        hdf5File.close()
+        n5File.close()
+
+        # Check the file.
+        hdf5File = h5py.File(self.testDataH5FileName, 'r')
+        n5File = h5py.File(self.testDataH5FileName, 'r')
+        h5_dataset = hdf5File[self.datasetInternalPath]
+        n5_dataset = n5File[self.datasetInternalPath]
+        assert h5_dataset.shape == self.dataShape
+        assert n5_dataset.shape == self.dataShape
+        assert numpy.all(h5_dataset[...] == self.testData.view(numpy.ndarray)[...])
+        assert numpy.all(n5_dataset[...] == self.testData.view(numpy.ndarray)[...])
+        hdf5File.close()
+        n5File.close()
+
+if __name__ == "__main__":
```

Comment:
please remove the `__main__` portion, we don't have those anymore (or will not, soon)

Reply:
this goes, of course, too, for the other test-files you touched :)